### PR TITLE
LocalPairing minor fixes. `ValidateConnectionString` method.

### PR DESCRIPTION
**Pull request by igor-sirotin**
Required by https://github.com/status-im/status-desktop/pull/9482

- Added `ValidateConnectionString` public method
- Added check if connection string is longer then 2 (to read `connectionStringID`)
- Other minor improvements and comments

### CODE REVIEW

File: mobile/status.go [hunk 1]
```diff
@@ -1003,7 +1003,7 @@ func GetConnectionStringForBootstrappingAnotherDevice(configJSON string) string
 //
 // Example: A mobile device (device with a camera) receiving account data from
 // a device with a screen (mobile or desktop devices)
-func InputConnectionStringForBootstrapping(cs, configJSON string) string {
+func InputConnectionStringForBootstrapping(cs string, configJSON string) string {
 	if configJSON == "" {
```
Comment (qfrank) [inline mobile/status.go]: can we not change this, i didn't see this is a problem ..

File: server/pairing/client.go [hunk 2]
```diff
@@ -267,7 +267,7 @@ func (c *Client) getChallenge() error {
 	return err
 }
 
-func StartUpPairingClient(backend *api.GethStatusBackend, cs, configJSON string) error {
+func StartUpPairingClient(backend *api.GethStatusBackend, cs string, configJSON string) error {
```
Comment (qfrank) [inline server/pairing/client.go]: similar to above

File: mobile/status.go (hunk 1 above)
Comment (igor-sirotin) [reply mobile/status.go]: Well, I just thought i's a typo. Was it intended to be no-type? If so, why?

Comment (qfrank) [reply mobile/status.go]: when you see something like `cs, configJSON string` in golang, it means cs has the same type(string) as configJSON, so it's not no-type here 🙂

**Review by richard-ramos — APPROVED** [review]

**Review by qfrank — APPROVED** [review]

**Pull request merged**